### PR TITLE
lntest: increase timeout for postgres backend

### PR DESCRIPTION
**Pull request by yyforyongyu**
A temp fix for the itest to unblock other PRs.

### CODE REVIEW

**Comment by yyforyongyu** [comment]
expecting failures - lets see how it builds

**Comment by yyforyongyu** [comment]
Two failures and this one is unexpected - pushed a new commit to fix it:
```
lnd_route_blinding_test.go:468: 
            Error Trace:    /Users/runner/work/lnd/lnd/itest/lnd_route_blinding_test.go:468
                                        /Users/runner/hostedtoolcache/go/1.22.3/arm64/src/runtime/asm_arm64.s:1222
            Error:          Received unexpected error:
                            rpc error: code = Canceled desc = context canceled
            Test:           TestLightningNetworkDaemon/tranche15/144-of-156/bitcoind/disable_introduction_node
```



Meanwhile the flake from `SendPayment` or `SendRoute` are known, **only if it's from a multi-hop setup test**, sth like:
```
harness_assertion.go:1078: 
            Error Trace:    /home/runner/work/lnd/lnd/lntest/harness_assertion.go:1078
                                        /home/runner/work/lnd/lnd/lntest/harness_assertion.go:1031
                                        /home/runner/work/lnd/lnd/lntest/harness.go:1877
                                        /home/runner/work/lnd/lnd/lntest/harness.go:1898
                                        /home/runner/work/lnd/lnd/itest/lnd_amp_test.go:134
                                        /home/runner/work/lnd/lnd/itest/lnd_amp_test.go:25
            Error:          Received unexpected error:
                            payment c24fd3ac669179416bf6bbe9a14e024df31b467eb4238273d54c15e89529f77f status, got IN_FLIGHT, want SUCCEEDED
            Test:           TestLightningNetworkDaemon/tranche11/101-of-156/bitcoind/sendpayment_amp_invoice/native_payaddr
            Messages:       timeout while waiting payment
```
Will make a note for all known test flakes.

**Review by ellemouton — APPROVED** [review]
noice

File: itest/lnd_route_blinding_test.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/btcsuite/btcd/btcec/v2"
 	"github.com/btcsuite/btcd/btcutil"
+	"github.com/go-errors/errors"
```
Comment (bhandras) [inline itest/lnd_route_blinding_test.go:12]: Intentional?

**Review by bhandras — APPROVED** [review]

File: itest/lnd_route_blinding_test.go (hunk 1 above)
Comment (yyforyongyu) [reply itest/lnd_route_blinding_test.go]: nice catch - weird this is the default import path on my IDE, now fixed.

**Pull request merged**